proposals/0390-noncopyable-structs-and-enums.md

```diff
@@ -909,14 +919,20 @@ func foo() {
 ```
 
 Mutable captures are subject to dynamic exclusivity checking like class
-properties are.
+properties are, and similarly cannot be consumed and reinitialized.
```

Comment:
I would add a blurb here explaining that this is necessary since otherwise we would break memory safety.